app/views/payments/new.html.haml

```diff
@@ -0,0 +1,14 @@
+.container
+  .row
+    .col-xs-6.col-xs-offset-3
+      %h1
+        Buy tickets for
+        = @total_amount_to_pay.symbol
```

Comment:
I think `humanised_money` and `#symbol` are used at a lot of places, is it okay if I replace by `Money#format` method that in an another PR?